lib/screens/v2/single_alert_widget.ex

```diff
@@ -1,3 +1,6 @@
+# Rename "ContextualAlert"? "EnrichedAlert"?
+# Just "Alert" - would that work? We'd need to rename the other module, then
```

Comment:
We could also go with Paul's suggested approach: do away with the protocol, and just define a new struct with the appropriate fields
(I think: `alert: Screens.Alerts.Alert.t()`, `location: SomeNewStruct???.t()`)

This would follow the same lines as the `Screens.V2.Departure` module/struct, which merges together a Prediction struct and a Schedule struct and provides functions that operate on that combined data.

The main question for that approach is, **can we define one common data structure for the "location context" of all of our alert-y widgets?**